samples/compose-browser/src/main/java/org/mozilla/samples/compose/browser/browser/BrowserScreen.kt

```diff
@@ -99,6 +99,9 @@ fun BrowserScreen(navController: NavController) {
                         onSuggestionClicked = { suggestion ->
                             store.dispatch(BrowserScreenAction.ToggleEditMode(false))
                             suggestion.onSuggestionClicked?.invoke()
+                        },
+                        onAutoComplete = { suggestion ->
+                            store.dispatch(BrowserScreenAction.UpdateEditText(suggestion.editSuggestion!!))
```

Comment:
We should probably also trigger text selection change, otherwise, the cursor will remain in the position before the text was autocompleted instead of being at the end.

Reply:
Yeah, I am planning to look at this separately when adding support for inline autocomplete in the toolbar.